helm/operator/values.yaml

```diff
@@ -1,54 +1,125 @@
-# Default values for minio-operator.
-
+###
+# Root key for Operator Helm Chart
 operator:
-  ## Setup environment variables for the Operator
-#  env:
-#    - name: MINIO_OPERATOR_DEPLOYMENT_NAME
-#      valueFrom:
-#        fieldRef:
-#          fieldPath: metadata.labels['app.kubernetes.io/name']
-#    - name: MINIO_CONSOLE_TLS_ENABLE
-#      value: "off"
-#    - name: CLUSTER_DOMAIN
-#      value: "cluster.domain"
-#    - name: WATCHED_NAMESPACE
-#      value: ""
-#    - name: MINIO_OPERATOR_RUNTIME
-#      value: "OpenShift"
+  ###
+  # An array of environment variables to pass to the Operator deployment.
+  # Pass an empty array to start Operator with defaults.
+  # For example:
+  #
+  # .. code-block:: yaml
+  #
+  #    env:
```

Comment:
List of env variables you can set in this section https://github.com/minio/operator/blob/master/docs/env-variables.md, probably we should refer to that doc, since it explains more clearly what each env var is for